### DISCUSSION TITLE
Is it possible to subscribe to updates on specific columns ?

### DISCUSSION THREAD

**Discussion by AaronNGray** [supabase/supabase#2039] [Questions] [answered]
Is it possible to subscribe to updates on specific columns ?

I.e. to generate a clause :- 

```CREATE TRIGGER name AFTER UPDATE OF column ...```

**Comment by w3b6x9** [comment, marked as answer]
@AaronNGray ~it's not possible to create a trigger for a specific column but you can create one for each updated row and filter out the column you're interested in.~

EDIT: Yep, it is possible in Postgres (i.e. `CREATE TRIGGER name AFTER UPDATE OF column ...`) but currently listening to a column is currently not possible in Realtime.

**Reply by AaronNGray** [reply]
I could do that for now and add a feature request to https://github.com/supabase/realtime as it is possible in PostgreSQL.

**Reply by w3b6x9** [reply]
@AaronNGray sounds good on the realtime feature request!

**Reply by AaronNGray** [reply]
https://github.com/supabase/realtime/issues/164
